### ISSUE TITLE
new cat-date label a11y

### ISSUE THREAD

**Issue by ddoe8** [haiilo/catalyst#523]
After having a look at the implementation of cat-date-inline, it was discussed that the a11y should be check regarding the readability of labels by a11y tools. This ticket is a reminder of that.
